src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -1179,5 +1462,29 @@ public void setOptType(OptType optType) {
         POLYGON_BOUND,
         NONE
     }
+    /**

```

Comment:
One more helper function here, this one to check programmatically if 2 CSG's are touching. I have found this helpfull to determine if i i will get an error when performing a CSG operation between 2 parts, if a collision is going to happen, and if shapes loaded from SVG's and fonts are holes or positive shapes.